tests/register.rs

```diff
@@ -25,32 +25,37 @@ impl UListener for FooListener {
     async fn on_error(&self, _err: UStatus) {}
 }
 
-#[test_case(test_lib::create_utransport_uuri(Some(0), 0, 0); "Publish / Notification register_listener")]
-#[test_case(test_lib::create_rpcserver_uuri(Some(0), 0); "RPC register_listener")]
-#[test_case(test_lib::create_special_uuri(0); "Special UUri register_listener")]
+#[test_case(&test_lib::new_uuri("src", 1, 1, 0x8000), None; "Listen to Publish")]
```

Comment:
Is there a way to listen to all point to point messages? I see ability to listen to all messages _of_ a given type. What about being able to express "all messages with a sink whose source authority_name is foo"?

That's one of the streamer use cases for connecting two authorities together over two transports.

Reply:
Just checked the uP-L1 Zenoh spec. Looks like this case may have been missed. Can we update the spec to clarify this is also possible and add this capability to the transport? (If I am reading it correctly it looks like registering to get all point to point messages as it was in 1.5.7 is not possible in the current implementation. Very possible I'm wrong and if so please point it out 😅)

Reply:
Is this what you expect? I added the uStreamer test cases in the tests now.
https://github.com/eclipse-uprotocol/up-transport-zenoh-rust/pull/40/commits/ea3a993497967418c31060ddeed91ea9fb632f5a#diff-c19a9c920bb76a5ab06a7d28f16df025d34c3a66697465b9e92c9df37ecdc0a3R36

Reply:
I think this is pretty close, left [a comment](https://github.com/eclipse-uprotocol/up-transport-zenoh-rust/commit/ea3a993497967418c31060ddeed91ea9fb632f5a#r143069220) on that test.